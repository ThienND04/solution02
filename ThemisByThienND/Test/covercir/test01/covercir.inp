5
684 36  338
403 153  332
591 232  323
116 112  33
595 131  99
