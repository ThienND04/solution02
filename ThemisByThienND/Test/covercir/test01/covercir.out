355
208
340
64
312
